.evergreen/config.yml

```diff
@@ -56,7 +55,6 @@ functions:
       params:
         working_dir: src
         script: |
-          ${PREPARE_SHELL}
           echo "TEST_TARGET=${TEST_TARGET}"
```

Comment:
you can either leave prepare_shell here, or convert to subprocess.exec, the lint is failing b/c PROJECT_DIRECTORY is missing, which would be added to the env by PREPARE_SHELL or we can use `add_expansions_to_env`